Controller/DefaultController.php

```diff
@@ -75,9 +75,10 @@ public function navigationAction($externalNetworkId, $seasonId, $options = array
         return $this->render(
             'CanalTPMttBundle:Default:navigation.html.twig',
             array(
-                'result'    => $result,
-                'seasonId'  => $seasonId,
-                'options'   => $options
+                'result'            => $result,
+                'seasonId'          => $seasonId,
+                'options'           => $options,
+                'navigationMode'    => (isset($options['navigationMode']) && $options['navigationMode'] != null) ? $options['navigationMode'] : 'routes'
```

Comment:
isset($options['navigationMode']) means $options['navigationMode'] exists and is not null
so $options['navigationMode'] != null is useless IMO